CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ option(BUILD_PROGRAMS "Build and install programs" ON)
 option(BUILD_EXAMPLES "Build and install examples" ON)
 option(BUILD_DOCS "Build and install doxygen documents" ON)
 option(WITH_STACK_PROTECTOR "Enable GNU GCC stack smash protection" ON)
+if(MINGW)
```

Comment:
This should be moved down to L75 if anything. Curious as to why you'd disable on MINGW? It shouldn't be adding the flag if the compiler doesn't support it.

Reply:
If it comes down to having to disable it for MingW the option shouldn't be exposed for MingW. There are many cleaner ways to do this.

Reply:
More info here: msys2/MINGW-packages#5803

This works for me. I you have clever solution i whould like to see it.

Reply:
-fstack-protector-strong should cause libssp to get linked, right? Wouldn't it only be an issue with __FORTIFY_SOURCE?

Anyway, the quickest band-aid is changing L76 of the old CMakeLists.txt from:
`if(NOT MSVC)`
to
`if(NOT MSVC AND NOT MINGW)`

Reply:
I've decided to remove this hack. No idea how to detect bugged MinGW. At least we have option to disable stack protector.